lib/xterm.plugin.bash.sh

```diff
@@ -0,0 +1,32 @@
+# script taken straight from bash-it with almost no adaptation
+# just commented 3 lines and added bash-preexec.sh into .bashrc
+
+#cite about-plugin
+#about-plugin 'automatically set your xterm title with host and location info'
+
+_short-dirname () {
+  local dir_name=`dirs +0`
+  [ "$SHORT_TERM_LINE" = true ] && [ ${#dir_name} -gt 8 ] && echo ${dir_name##*/} || echo $dir_name
```

Comment:
```suggestion
  [ "$SHORT_TERM_LINE" = true ] && [ ${#dir_name} -gt 8 ] && echo "${dir_name##*/}" || echo "$dir_name"
```

We should quote them to protect the execution from `shopt -s failglob`. There are also similar cases in other lines.

Reply:
same as above.

also applied for the next line.